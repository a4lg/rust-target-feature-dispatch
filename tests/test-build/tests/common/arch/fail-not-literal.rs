pub const VALUE: i32 = target_feature_dispatch::target_feature_dispatch! {
    // ERROR: (()) is not an architecture.
    if (()) {
        (1)
    } else {
        0
    }
};
fn main() {}
