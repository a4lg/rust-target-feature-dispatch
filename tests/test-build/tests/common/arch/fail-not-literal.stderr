error: expected unsuffixed literal, found `(`
 --> tests/common/arch/fail-not-literal.rs:3:8
  |
3 |     if (()) {
  |        ^

error[E0308]: mismatched types
 --> tests/common/arch/fail-not-literal.rs:4:10
  |
4 |         (1)
  |          ^ expected `()`, found integer
